zinc/src/main/scala/sbt/internal/inc/IncrementalCompilerImpl.scala

```diff
@@ -176,7 +176,7 @@ class IncrementalCompilerImpl extends IncrementalCompiler {
 
         val msg =
           s"""## Exception when compiling $numberSources to $outputString
-             |${e.getMessage}
+             |${e.toString}
```

Comment:
Why this change? Is it to see the message of a nested exception that bubbled up?

Reply:
We can sometimes face an exception with useless mesage like `null` or something. Sometimes what actually happened is explained in the type of the exception. Now we will get "ExceptionClass: exceptionMessage". I don't remember the specific exceptions I was facing but this helped me.